readme.md

```diff
@@ -112,11 +110,6 @@ For more information, please see the Google Analytics Measurement Protocol
 
 Tests can be run using `pub run test`.
 
-## Issues and bugs
-
-Please file reports on the
-[GitHub Issue Tracker](https://github.com/dart-lang/usage/issues).
-
 ## License
```

Comment:
Do any other README we own bother to mention the license specifically?

Reply:
ack